packages/table/src/common/internal/scrollUtils.ts

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Really want to get into the practice of taking utility code out of `table.tsx`. I'd also like to avoid putting all utility code into one huge `Utils` file if possible. Thus, here's a new `scrollUtils` file.

Reply:
that's cool, if you add more do you think it's useful to keep track of an `index.ts` that re-exports? it's a tradeoff between maintenance of index.ts and simplifying import lines.

Reply:
I'd be into that. 👍